center_of_mass_m:
- 0.015976
- 0.000351
- 0.040641
mass_kg: 0.319596
static_force_offset_N:
- -0.62875
- -0.13113
- -3.02676
static_torque_offset_Nm:
- -0.02941
- -0.65378
- -0.0238

T_robot_to_mocap:
- [ 0.999856,  0.016928, -0.000914,  0.112686]
- [-0.016925,  0.999848,  0.004154,  1.893302]
- [ 0.000984, -0.004138,  0.999991,  0.756350]
- [ 0.000000,  0.000000,  0.000000,  1.000000]

T_ee_to_tool:
- [1.000000, 0.000000, 0.000000, 0.000000]
- [0.000000, 1.000000, 0.000000, 0.000000]
- [0.000000, 0.000000, 1.000000, 0.237000]
- [0.000000, 0.000000, 0.000000, 1.000000]

T_sensor_to_ee:
- [1.000000, 0.000000, 0.000000, 0.000000]
- [0.000000, 1.000000, 0.000000, 0.000000]
- [0.000000, 0.000000, 1.000000, 0.000000]
- [0.000000, 0.000000, 0.000000, 1.000000]

T_cam_to_robot:
- [0.99910564, -0.03531599, 0.02325316, -0.49901560]
- [-0.04052208, -0.64260646, 0.76512410, -0.63141464]
- [-0.01207848, -0.76538207, -0.64346282, 0.48368062]
- [0.00000000, 0.00000000, 0.00000000, 1.00000000]